hyp3lib/rtc2color.py

```diff
@@ -43,26 +68,30 @@ def rtc2color(fullpolFile, crosspolFile, threshold, geotiff, cleanup=False,
   print('Estimated Total RAM usage = {} GB'.format(size*16))
 
   # Read full-pol image
-  print('Reading full-pol image (%s)' % fullpolFile)
+  print('Reading full-pol image (%s)' % out_tif)
   data = fullpol.GetRasterBand(1).ReadAsArray()
   cp = (data[:rows, :cols]).astype(np.float16)
   data = None
   cp[np.isnan(cp)] = 0
+  # FIXME: do we get negative powers from RTC?
   cp[cp < 0] = 0
   if cleanup == True:
     cp[cp < 0.0039811] = 0
+  # FIXME: Should this be applied *before* the above cleanup?
   if amp == True:
     cp = cp*cp
 
   # Read cross-pol image
-  print('Reading cross-pol image (%s)' % crosspolFile)
+  print('Reading cross-pol image (%s)' % crosspol_tif)
   data = crosspol.GetRasterBand(1).ReadAsArray()
   xp = (data[:rows, :cols]).astype(np.float16)
   data = None
   xp[np.isnan(xp)] = 0
   xp[xp < 0] = 0
   if cleanup == True:
-    xp[xp < 0.0039811] = 0
+      # FIXME: Is this the correct value? 0.0039811 ~= pow(10.0, -24.0 / 10.0),
+      #        or our default threshold
+      xp[xp < 0.0039811] = 0
```

Comment:
The cleanup threshold we're using is equal to our default `-24 db` option on the website -- that seems like a high cleanup for one, and two is unnecessary if it is correct. *Note: done for both `cp` and `xp`*

For the granule in the user story, this would eliminate ] in `cp` (teals and yellows):
![image](https://user-images.githubusercontent.com/7882693/84719311-81884c00-af27-11ea-9777-e640e16357c8.png)

 and in `xp`:
![image](https://user-images.githubusercontent.com/7882693/84719320-8816c380-af27-11ea-8014-e1045a37455f.png)

Reply:
This cleanup is correct for amplitude data, but should be `-48 db` for power data.